docs/assembly.rst

```diff
@@ -72,7 +72,7 @@ Example
 -------
 
 The following example provides library code to access the code of another contract and
-load it into a ``bytes`` variable. This is not possible with "plain Solidity" and the
```

Comment:
Are you sure? This is meant to mean "... to access ... and _to_ load it...". What do you think, @ChrisChinchilla ?